airbyte-integrations/singer/exchangeratesapi/source/Dockerfile

```diff
@@ -1,21 +1,20 @@
 FROM airbyte/integration-base-singer:dev
 
-RUN apt-get update && apt-get install -y jq
-
-COPY spec.json /airbyte/exchangeratesapi-files/spec.json
-
-WORKDIR /airbyte/source_exchangeratesapi_singer
-
-COPY source_exchangeratesapi_singer/__init__.py ./source_exchangeratesapi_singer/__init__.py
-COPY source_exchangeratesapi_singer/source_exchangeratesapi_singer.py ./source_exchangeratesapi_singer/source_exchangeratesapi_singer.py
-
-COPY setup.py .
-RUN pip install .
-
-WORKDIR /airbyte
+RUN apt-get update && apt-get install -y \
+    jq \
+    && rm -rf /var/lib/apt/lists/*
 
+ENV CODE_PATH="source_exchangeratesapi_singer"
```

Comment:
could put this below and reuse `AIRBYTE_IMPL_MODULE`

Reply:
I wanted to do but in it current form because we don't have a good convention so it is dangerous to do so, both AIRBYTE_IMPL_MODULE and 
AIRBYTE_IMPL_PATH could could contain module info. eg:
```
AIRBYTE_IMPL_MODULE=module
AIRBYTE_IMPL_PATH=submodule.Source
```
```
AIRBYTE_IMPL_MODULE=module.submodule
AIRBYTE_IMPL_PATH=Source
```

Both these options are valid.

so you might endup not copying everything.